src/FileMigrationHelper.php

```diff
@@ -56,27 +58,40 @@ public function run($base = null)
         Environment::increaseTimeLimitTo();
         Environment::increaseMemoryLimitTo();
 
+        /** @var LoggerInterface $logger */
+        $logger = Injector::inst()->get(LoggerInterface::class);
+
         // Loop over all files
-        $count = 0;
+        $i = $migrated = 0;
```

Comment:
Minor detail, I prefer `$count` > `$i`. But `$processedCount` and `$migratedCount` would be even better.